server/game/cards/03-WC/Memolith.js

```diff
@@ -11,7 +11,7 @@ class Memolith extends Card {
                     card.location === 'hand' ||
                     (card.parent === context.source && card.location === 'grafted'),
                 gameAction: ability.actions.conditional({
-                    condition: (context) => context.target.location === 'hand',
+                    condition: (context) => context.target && context.target.location === 'hand',
```

Comment:
target should never be undefined here, if it is, I'd rather it blow up because it's a bug in the engine